### DISCUSSION TITLE
docker windows surviving reboot

### DISCUSSION THREAD

**Discussion by cwm12345** [mrlt8/docker-wyze-bridge#831] [Q&A]
Hi, I've searched and searched and tried and tried but just cannot sort out how to have Docker (specifically the parts installed with Docker Desktop) survive a reboot so that the wyze-bridge container restarts. I feel like I am probably missing something super basic since my searching isn't producing an obvious answer.

Would be greatful for any pointers on how to setup the wyze bridge container on a windows 11 machine so that it restarts after a reboot. In my case the docker stuff has to be manually restarted and then the container starts right up -- so my issue is docker, not the specific container. Thanks in advance.

**Comment by cheme75** [comment]
There’s an option in settings to start with windows. On my win 10 pro machine, it seems to start up all the background tasks but the desktop doesn't start until manually starting it from the shortcut. Meanwhile it seems the container won’t start until the desktop is running, although I haven’t tried to access the bridge before starting the docker desktop so not sure. I know on a Linux docker that lacks the desktop, the container starts up in docker, just no desktop to view its log or stats.
